storage/backend/ceph/internals.go

```diff
@@ -6,14 +6,30 @@ import (
 	"regexp"
 	"strings"
 
+	"encoding/json"
+
 	"github.com/contiv/volplugin/storage"
 
 	log "github.com/Sirupsen/logrus"
 )
 
 func (c *Driver) mapImage(do storage.DriverOptions) (string, error) {
-	blkdev, err := exec.Command("rbd", "map", do.Volume.Name, "--pool", do.Volume.Params["pool"]).Output()
-	device := strings.TrimSpace(string(blkdev))
+	var outputdata map[string]interface{}
```

Comment:
Instead of a `map[string]interface{}`, I think it might be better to define `outputdata` to be a struct, this will avoid the dictionary lookup and type assertion below at line 29 and hence a chance of panic.

I am not sure what the json output looks like, but assuming it is a array of volume infos, we might be able to do something like this:

```
var outputData struct {
    volumes []struct {
        Name string `json:"name"`
    } `json:"volumes"`
}

json.Unmarshal(output, &outputData)
...
...

if outputdata[i].Name != do.Volume.Name {
...
}
```